flink-yarn/pom.xml

```diff
@@ -227,6 +227,27 @@ under the License.
 					</execution>
 				</executions>
 			</plugin>
+			<!-- Write classpath of flink-yarn to a file, so that the yarn tests can use it as their classpath
```

Comment:
why aren't we using the classpath of flink-yarn-tests?

Reply:
I don't remember / maybe there's no reason. I am trying to use `flink-yarn-tests`.

Reply:
It seems to work.